activities/hinks.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+"""
+Instructions
+- Functions intended to run for each scan
+  need to be named run_xxxxx
+- Do not modify the parameters of the run_xxx function
+  unless you know what you are doing
+  see optional parameters:
+  https://github.com/tangible-landscape/grass-tangible-landscape/wiki/Running-analyses-and-developing-workflows#python-workflows
+- All gs.run_command/read_command/write_command/parse_command
+  need to be passed env parameter (..., env=env)
+"""
+
+import grass.script as gs
+
+
+def run_ponds(scanned_elev, env, **kwargs):
+    repeat = 20
```

Comment:
this is too many times, 2 or 3 times should be fine here

Reply:
Thank you, that is very helpful feedback. I changed the repeat variable to be 3 in the updated pull request.